### ISSUE TITLE
Pointer Lock Unadjusted Movement Option.

### ISSUE THREAD

**Issue by JamesHollyer** [mozilla/standards-positions#448]
## Request for Mozilla Position on an Emerging Web Specification

* Specification Title: Pointer Lock Unadjusted Movement
* Specification or proposal URL: https://github.com/w3c/pointerlock/pull/49
* Caniuse.com URL (optional): 
* Bugzilla URL (optional): 
* Mozillians who can provide input (optional): probably @smaug---- 

### Other information
Issue was discussed here: https://github.com/w3c/pointerlock/issues/36

Many new gaming platforms are coming out for the web and this is a highly requested feature from those platforms.

**Comment by marcoscaceres** [comment]
Gentle ping... wondering if anyone from Mozilla has a few spare cycles to give some input here? 🙏 The [proposal](https://github.com/w3c/pointerlock/pull/49) is more or less ready to go, so the Editors would really appreciate some input from Mozilla. Maybe just a nod that there is nothing particular abhorrent and that Mozilla would be fine for that proposal to proceed.

**Comment by NoahBPeterson** [comment]
https://github.com/w3c/pointerlock/pull/99

This is an official web standard now, is this enough to get a standard position from Mozilla so this can be worked on?

@zcorpan If the blocker here is developer time, let me know and I will make a PR. Otherwise, it seems that the blocker is a position from the standards committee.

- Specification Title: Pointer Lock Unadjusted Movement
- Specification or proposal URL: https://github.com/w3c/pointerlock/pull/99 / https://w3c.github.io/pointerlock/#pointerlockoptions-dictionary
- Caniuse.com URL (optional): https://caniuse.com/mdn-api_element_requestpointerlock_options_unadjustedmovement_parameter
- Bugzilla URL (optional): https://bugzilla.mozilla.org/show_bug.cgi?id=1829401
- Mozillians who can provide input (optional)

**Comment by zcorpan** [comment]
I agree that disabling mouse acceleration is needed for some games (in particular FPS games). The API seems reasonable to me. I suggest position: positive.

**Comment by zcorpan** [comment]
> @zcorpan If the blocker here is developer time, let me know and I will make a PR. Otherwise, it seems that the blocker is a position from the standards committee.

No need for a PR to resolve this standards-position. If you mean you wish to contribute to Gecko for this feature, I'd suggest commenting in the bug. Thanks!
